### DISCUSSION TITLE
Anyway to set timelimit for sessions?

### DISCUSSION THREAD

**Discussion by nikhilraojl** [maxcountryman/axum-login#92] [Q&A] [answered]
Is there a way to have something like sesison timeout? The cookie set after login should be invalid for only for certain time

**Comment by maxcountryman** [comment, marked as answer]
Have you looked at the [session TTL](https://docs.rs/axum-sessions/latest/axum_sessions/struct.SessionLayer.html#method.with_session_ttl)?

**Comment by nikhilraojl** [comment]
Thank you, this is what I was looking for.

Sorry for posting the question here then, I need to read docs better. I was searching for terms like timeout, time and such but did not get any results in `axum_sessions` docs and just posted the question here 🙇

**Reply by maxcountryman** [reply]
Glad that was what you were looking for.
